uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/viewmodel/RealFeedViewModel.kt

```diff
@@ -40,6 +46,16 @@ class RealFeedViewModel @Inject constructor(
         }
     }
 
+    private val mediaPlayer by lazy {
+        MediaPlayer().apply {
+            setAudioAttributes(
+                AudioAttributes.Builder()
+                    .setLegacyStreamType(AudioManager.STREAM_MUSIC)
+                    .build()
+            )
+        }
+    }
```

Comment:
Thanks for implementing it to make it playable! 👍 
If you put the processing related to Player in FeedViewModel, it seems that it will be too large, so it seems better to separate it into another ViewModel. There is no need to respond with this pull request.

Reply:
I understand. Thanks for your advice!